apps/web/src/constants/cacheTimes.ts

```diff
@@ -0,0 +1,18 @@
+const ONE_MINUTE = 60
+const ONE_HOUR = ONE_MINUTE * 60
+const ONE_DAY = ONE_HOUR * 24
+
+export const CACHE_TIMES = {
+  DAO_INFO: {
+    maxAge: ONE_MINUTE,
+    swr: ONE_DAY,
+  },
+  TOKEN_INFO: {
+    maxAge: ONE_MINUTE,
+    swr: ONE_DAY,
+  },
+  PROPOSAL: {
+    maxAge: ONE_HOUR,
+    swr: ONE_DAY,
```

Comment:
can we test that active proposals with active set of votes are correctly handled? we're re-fetching client side so it should be fine i think. i'm wondering if swr value should be a whole day though. any particular reasoning for a whole day?

Reply:
it should refetch on client like you mentioned but I think I might rework to do some lighter caching for active proposals and cache more aggressively for failed / succeeded / canceled ones  as those should never change. 

What do you think about one minute maxAge / swr one hour for active proposals and one Day maxAge / swr one week on inactive proposals.

my thinking with the long swr for inactive proposals is they should not change at all and it will make older proposals that get viewed semi often much faster to load. agreed that one hour / one day is too aggressive for active proposals. let me know what you think

Reply:
aha - agree that active should should have a shorter lifespan and that non active proposals can be longer cached 👍